src/commands/templates/discover.js

```diff
@@ -17,25 +17,74 @@ const inquirer = require('inquirer')
 const { TEMPLATE_PACKAGE_JSON_KEY, readPackageJson } = require('../../lib/npm-helper')
 const { getTemplates } = require('../../lib/template-registry-helper')
 const aioLogger = require('@adobe/aio-lib-core-logging')('@adobe/aio-cli-plugin-app-templates:templates:discover', { provider: 'debug' })
+const loadConfig = require('@adobe/aio-cli-lib-app-config')
+const env = require('@adobe/aio-lib-env')
+const LibConsoleCLI = require('@adobe/aio-cli-lib-console')
 
 class DiscoverCommand extends BaseCommand {
+  async run () {
+    const { flags } = await this.parse(DiscoverCommand)
+    const spinner = ora()
+
+    try {
+      const searchCriteria = {
+        statuses: ['Approved']
+      }
+      const orderByCriteria = {
+        [flags['sort-field']]: flags['sort-order']
+      }
+      spinner.start()
+      const templates = await getTemplates(searchCriteria, orderByCriteria)
+      aioLogger.debug(`Retrieved templates: ${JSON.stringify(templates, null, 2)}`)
+      spinner.stop()
+
+      if (flags.interactive) {
+        return this.__install(templates)
+      } else {
+        return this.__list(templates)
+      }
+    } catch (error) {
+      spinner.stop()
+      this.error('Oops:' + error)
+    }
+  }
+
   async __install (templates) {
     const spinner = ora()
 
     const packageJson = await readPackageJson()
     const installedTemplates = packageJson[TEMPLATE_PACKAGE_JSON_KEY] || []
     aioLogger.debug(`installedTemplates: ${JSON.stringify(installedTemplates, null, 2)}`)
 
+    await this.login()
+    const apiKey = env.getCliEnv() === 'prod' ? 'aio-cli-console-auth' : 'aio-cli-console-auth-stage'
+    const consoleCLI = await LibConsoleCLI.init({ accessToken: this.accessToken, env: env.getCliEnv(), apiKey: apiKey })
+
+    const appConfig = loadConfig({})
+    let orgId = appConfig?.aio?.project?.org?.id
+    if (orgId == null) {
```

Comment:
If `org` is not defined, for example, `orgId` will be `undefined`, not `null`.
```suggestion
    if (!orgId) {
```